CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixes
 * Fixed crash when adding classes containing an `ObjectId` as primary key to the schema (Issue [#7189](https://github.com/realm/realm-java/issues/7189), since v10.0.0).
```

Comment:
```suggestion
* Fixed crash when adding classes containing an `ObjectId` as primary key to the schema. (Issue [#7189](https://github.com/realm/realm-java/issues/7189), since v10.0.0).
```